tests/test_dir_recursion.py

```diff
@@ -36,13 +32,11 @@ def setUp(self):
     def tearDown(self):
         shutil.rmtree("a", ignore_errors=True)
 
-    # pylint: disable-next=invalid-name
-    def assertEqualContent(self, file1, content_expected):
+    def assertEqualContent(self, file1, content_expected):  # noqa: N802
```

Comment:
I think we had this before, could add it to the configuratoin as well, maybe?

https://docs.astral.sh/ruff/settings/#lint_pep8-naming_extend-ignore-names

If you want to keep it inline, is it possible to append the name of the rule?

Reply:
I like that since it avoids all the explicit `# noqa` in the code which is a bad habit (e.g., like you mentioned since it needs extra docs).